api/src/main/java/org/openmrs/liquibase/ChangeLogVersions.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.liquibase;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Defines which Liquibase snapshot and update change logs are available at all. 
+ * 
+ * The information provided by this class needs to be updated when new Liquibase snapshots and updates
+ * are added to openmrs-api resources.
+ */
+public class ChangeLogVersions {
+	/**
+	 * This definition of Liquibase snapshot versions needs to be kept in sync with the actual change log files
+	 * in openmrs-core/api/src/main/resources/liquibase/snapshots/core-data 
+	 * and openmrs-core/api/src/main/resources/liquibase/snapshots/schema-only.
+	 * 
+	 * If the actual change log files and this list get out of sync, org.openmrs.liquibase.ChangeLogVersionsTest fails. 
+	 */
+	private static final List<String> SNAPSHOT_VERSIONS = Arrays.asList(
+		"1.9.x",
+		"2.1.x",
+		"2.2.x",
+		"2.3.x"
+	);
+
+	/**
+	 * This definition of Liquibase update versions needs to be kept in sync with the actual change log files
+	 * in openmrs-core/api/src/main/resources/liquibase/updates. 
+	 *
+	 * If the actual change log files and this list get out of sync, org.openmrs.liquibase.ChangeLogVersionsTest fails. 
+	 */
+	private static final List<String> UPDATE_VERSIONS = Arrays.asList(
+		"1.9.x",
+		"2.0.x",
+		"2.1.x",
+		"2.2.x",
+		"2.3.x",
+		"2.4.x"
+	);
+
+	public List<String> getSnapshotVersions() {
+		return SNAPSHOT_VERSIONS;
+	}
+
+	public List<String> getUpdateVersions() {
+		return UPDATE_VERSIONS;
+	}
```

Comment:
Absolutely! The abundance of static methods on OpenMRS makes it really hard to test and I tend to avoid static where possible.

Reply:
I suppose that's a reasonable point. I hate static methods as well, but having a class just to return some static lists is weird. I suppose if it can't be helped, it can't be helped.